### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ikoote1**
###SETUP CHANGES:
- Created a new directory for your reusable components: /components.
- Installed react.
- Installed Routers.
- Added 2 routes in your application which contain
 a book list and form that allows for creating a new book,
 and categories.

- Included a navigation component that includes links to the 2 routes I created.

- Checked and cleared errors.

- Added some styling.

- Pushed and created a pull request.

Thanks for reviewing my work.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>ikoote</h1>
+);
```
Comment (afanimartin) [inline src/components/Categories.js:3]: - This component should have the `Button` element with the text: `check status`. Please the required changes.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const Books = () => (
+  <div><p>ikooterhehfhdhf</p></div>
```
Comment (afanimartin) [inline src/components/Books.js]: - This component should contain the following components:
  1- <BookList> - Renders a list of books using the 
       <Book> component
  2- <Book> - Takes the book data as a prop and renders the title and the author. Has a button for removing books. The button wouldn't remove the books yet, this is just a placeholder for now
   3- <NewBookForm> - Renders a form with inputs for the title and the author of the book. Has a button to submit the button wouldn't add the book yet, this is just a placeholder for now.

Please make the above changes and resubmit for review.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @ikoote1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Books and Categories components implemented as required
- Professional readme template was used
- Navigation links added to the main component and `react-router-dom` package installed as required.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — APPROVED** [review]
Hi @ikoote1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Good job using react router to route through your components!👏👏👏
- [x] Great job Adding the BOOK and CATEGORIES components!👏👏👏

## Optional suggestions

**- [Optional] Please in all future projects, make sure the name of your branch is the name of a feature or action you are implementing on the branch
- [Optional] Please in all future projects, kindly make sure the PR title describes the changes implemented**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**